README.md

```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
 $ git worktree add vsc
 $ git worktree add wip
 ```
+
+## Commits with worktrees
+
+Pushing/pulling changes in worktrees is no different than in a regular repo. As soon as you change directory into a worktree folder, you can work as if you were on a regular repo. There will be an active branch, you can create/checkout other branches and commit to any branch as usual.
+
+### Unreviewed branches
+
+Branches such `wip` do not require PRs and reviews to push changes.
+
+1. Enter the target worktree/branch
+```
+$ cd vsc-software-stack/wip
+```
+2. Add and commit the files affected by this change
```

Comment:
Step 2 is to update to latest version of `wip` branch first (`git pull origin wip`)

Reply:
fixed in https://github.com/vscentrum/vsc-software-stack/pull/111/commits/af209f37e1c5ffb9ca1ccbd6b1475b5d8a1d7380